docu/Concepts/Snippets/1-Blockchain/Roles.md

```diff
@@ -0,0 +1,53 @@
+# Roles
+User Roles also handled by blockchain and node servers
+
+My Goal is to save not only the gradido transactions in blockchain but also 
+who is allowed to create new gradidos,
+who allow joining new user to the group,
+who allow connect to another group and 
+how the community decide which one is allowed to do this things.
+
+## Why?
+If this would be handled only by community-server everyone could be easly 
+overwrite this rules by using a modified client to send his transactions direct over
+hedera or iota bypassing the community-server rules. 
```

Comment:
Aussage: Leute können auf die Blockchain schreiben, indem sie ihren eigenen Client benutzen.
Aussage: Wenn wir die Rechte auf der Blockchain abbilden, dann können sie das nicht mehr.

Frage: Warum können sie das nicht mehr? Die Rechte stehen dann auf der Blockchain, wie verhindert das, dass Nutzer falsche Transaktionen in das Netzwerk geben?
Frage: Wäre nicht ein Konsensusalgorithmus die Stelle, an der entschieden werden sollte, was eine legale Transaktion ist?
Frage: Ist das dann noch kompatibel mit Blockchain, die ja Egalität schaffen soll?

Reply:
Was ist DHT?

Reply:
Die Nutzer können falsche Transaktionen ins Netz geben, sie werden dann aber von den Node-Servern nicht angenommen oder als ungültig gespeichert.
Eigentlich hast du Recht Ulf, besser wäre das schon im Konsensus-Algorithmus zu entscheiden, dann haben wir aber noch mehr Arbeit bis wir soweit sind, das wir eine Transaktion von einer Community zu einer anderen senden kann. 
Dafür Iota zu verwenden ist eine Abkürzung, das wir erstmal etwas haben, bevor wir anfangen da unsere eigene Entwicklung zu machen. Weil das ist wirklich aufwändig.